contracts/spec_interfaces/ICommittee.sol

```diff
@@ -4,40 +4,59 @@ import "./IContractRegistry.sol";
 
 /// @title Elections contract interface
 interface ICommittee {
+	// State events
```

Comment:
We used to have a `TopologyChanged` event with orbs addresses and IPs for the entire topology (committee + standbys), emitted on any change to topology (ip change, orbs addr chagne, validator added/removed). Do we still have this?